oteapi/strategies/download/https.py

```diff
@@ -9,25 +8,30 @@
 from oteapi.plugins import StrategyFactory
 
 if TYPE_CHECKING:
-    from typing import Any, Dict, Optional
+    from typing import Any, Dict
 
     from oteapi.models import ResourceConfig
 
 
 @dataclass
 @StrategyFactory.register(("scheme", "http"), ("scheme", "https"))
 class HTTPSStrategy:
-    """Strategy for retrieving data via http."""
+    """Strategy for retrieving data via http.
+
+    **Registers strategies**:
+
+    - `("scheme", "http")`
+    - `("scheme", "https")`
+
+    """
 
     resource_config: "ResourceConfig"
 
-    def initialize(
-        self, session: "Optional[Dict[str, Any]]" = None
-    ) -> "Dict[str, Any]":
-        """Initialize"""
+    def initialize(self, **_) -> "Dict[str, Any]":
+        """Initialize."""
         return {}
 
-    def get(self, session: "Optional[Dict[str, Any]]" = None) -> "Dict[str, Any]":
+    def get(self, **_) -> "Dict[str, Any]":
```

Comment:
I am not fully comfortable with this use of anonymous arguments. The get() and other methods should expect one optional `session` parameter, not an arbitrary number of anonymous keyword arguments.

Reply:
It's fair enough. Discussing this offline, we have arrived at the conclusion of keeping in `session` explicitly - even if not used - to ensure the signature is clear for external strategy developers as well.
The pylint disable statements must be returned also.